modules/ossm-config-control-plane-infrastructure-node-console.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-control-plane-infrastructure-node-console_{context}"]
+= Configuring all control plane components to run on infrastructure nodes using the web console
+
+Perform this task if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, will run on infrastructure nodes. 
+
+If the control plane will run on a worker node, skip this task.
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You must have an account with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
```

Comment:
Same question as in my previous comment.

Reply:
done